src/utils/notifyError.ts

```diff
@@ -1,7 +1,11 @@
 import { window } from 'vscode'
+import { ext } from '../extensionVariables'
+import { genericMessage } from '../messages/messages'
 
-import { errorMessage } from '../messages/messages'
+export function notifyError(error: unknown, failedSourceErrorMessage: string) {
+  window.showErrorMessage(`${failedSourceErrorMessage} ${genericMessage.checkOutputChannel}`)
 
-export function notifyError(error: unknown) {
-  window.showErrorMessage(error instanceof Error ? error.message : errorMessage.unexpectedError)
+  ext.outputChannel.appendLine(failedSourceErrorMessage)
+  ext.outputChannel.appendLine(`${error}`)
+  ext.outputChannel.show()
 }
```

Comment:
We have improved this method quite a bit and I think it makes sense to move `logger.error()` here to avoid duplicating it next to every `notifyError()`.

Also I'd make few changes:
```ts
export function notifyError(error: any, customMessage: string) {
  const message = customMessage || error.message || 'Unexpected error'

  logger.error(error, message)
  window.showErrorMessage(genericMessage.errorNotification(customMessage))
  ext.outputChannel.appendLine(`${message}: ${error}`)
  ext.outputChannel.show()
}
```